languages/ruby/bitwarden_sdk/bitwarden-sdk-ruby.gemspec

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'lib/version'
+
+Gem::Specification.new do |spec|
+  spec.name = 'bitwarden-sdk-ruby'
```

Comment:
```suggestion
  spec.name = 'bitwarden-sdk'
```

Reply:
changed, as well as all over the gem files